### PR TITLE
First draft of README.adoc file

### PR DESCRIPTION
**Pull request by bredamc**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    [-v <level>] [--verbosity <level>]
+    Verbosity level for app. Not used currently.
+
+    [--version]
+    If specified, print version number and exit.
+
+
+Getting inline help is:
+
+    docker run --rm fnndsc/pl-heudiconv heudiconv --man
```
Comment (grdryn) [inline README.adoc:62]:
```suggestion
    docker run --rm quay.io/rh-impact/pl-heudiconv pl-heudiconv --man
```

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+### Run
+
+You need to specify input and output directories using the `-v` flag to `docker run`.
+
+    docker run --rm -u $(id -u)                             \
+        -v $(pwd)/in:/incoming -v $(pwd)/out:/outgoing      \
+        fnndsc/pl-heudiconv heudiconv                        \
```
Comment (grdryn) [inline README.adoc:70]:
```suggestion
        quay.io/rh-impact/pl-heudiconv pl-heudiconv                        \
```

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,88 @@
+# HeuDiConv
+
+[%autowidth, cols="1,1,1", frame=none, grid=none]
+|===
+a|
+image::https://img.shields.io/docker/v/fnndsc/pl-heudiconv?sort=semver[https://hub.docker.com/r/fnndsc/pl-heudiconv]
+a|
+image::https://img.shields.io/github/license/fnndsc/pl-heudiconv[https://github.com/FNNDSC/pl-heudiconv/blob/main/LICENSE]
```
Comment (grdryn) [inline README.adoc:8]:
```suggestion
image::https://img.shields.io/github/license/rh-impact/pl-heudiconv[https://github.com/rh-impact/pl-heudiconv/blob/main/LICENSE]
```

File: README.adoc [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+a|
+image::https://github.com/FNNDSC/pl-heudiconv/actions/workflows/ci.yml/badge.svg[https://github.com/FNNDSC/pl-heudiconv/actions/workflows/ci.yml]
```
Comment (grdryn) [inline README.adoc:10]:
```suggestion
image::https://github.com/rh-impact/pl-heudiconv/actions/workflows/ci.yml/badge.svg[https://github.com/rh-impact/pl-heudiconv/actions/workflows/ci.yml]
```

File: README.adoc [hunk 5]
```diff
@@ -0,0 +1,88 @@
+# HeuDiConv
+
+[%autowidth, cols="1,1,1", frame=none, grid=none]
+|===
+a|
+image::https://img.shields.io/docker/v/fnndsc/pl-heudiconv?sort=semver[https://hub.docker.com/r/fnndsc/pl-heudiconv]
```
Comment (grdryn) [inline README.adoc:6]:
```suggestion

```

I think this one can be removed for now, I don't think there's an equivalent for quay.io yet.

**Comment by bredamc** [comment]
@grdryn Some questions about this plugin, to help me to populate the README doc:

1. What input formats can be used? Is it just DICOM files? Do DICOM files always have the `.dcm` filetype?
2. What is the output format? NIfTI, BIDS, other?
3. Can the user customise the output format, and if so, how?
4. Is there any other information that we should include for the user?

Many thanks :)

**Comment by grdryn** [comment]
> 1. What input formats can be used? Is it just DICOM files? Do DICOM files always have the .dcm filetype?

I believe it's just DICOM files, but I'm unsure. I've only seen ones that have the `.dcm` extension, but I've only seen about 7, so that might not a very big sample :see_no_evil: Maybe we don't need to mention that? According to the [HeuDiConv docs](https://heudiconv.readthedocs.io/en/latest/index.html) it sounds like it will ignore everything in a directory apart from the DICOM files.

> 2. What is the output format? NIfTI, BIDS, other?

I think it depends on the options that are specified. It seems to give directory trees; the structure and contents of which, seem to depend on the inputs and the options.

> 3. Can the user customise the output format, and if so, how?

The underlying `heudiconv` tool does expose options to customise things somewhat, but those haven't been made available in this plugin _yet_. I'm also not familiar with any of the files that it exposes based on any of the options.

> 4. Is there any other information that we should include for the user?

I think the most important thing to include would be a link to the docs for the underlying `heudiconv` tool: https://heudiconv.readthedocs.io/en/latest/index.html
I'd be hesitant to try to reproduce any of that, since we (or at least I) don't have the expertise to know if what we're saying makes sense.

Maybe one thing we could mention is that this aims to be just a wrapper around the `heudiconf` CLI tool, to make it available as a ChRIS plugin. In `heudiconv` where the input and output directories are named arguments, ChRIS plugins expect them to be positional arguments. As a result of this translation, some other arguments may be missing or structured/named differently.

**Review by grdryn — APPROVED** [review]
Thanks @bredamc, great job! I'll merge this now, and we can iteratively improve on it in future PRs :100:

**Comment by bredamc** [comment]
Many thanks for your replies (and for merging this PR!) @grdryn -- I'll make some more doc updates in a new PR and assign you as reviewer :)

**Pull request merged**